2024-05-19 16:47:56,368 INFO    MainThread:1428650 [wandb_setup.py:_flush():76] Current SDK version is 0.16.5
2024-05-19 16:47:56,368 INFO    MainThread:1428650 [wandb_setup.py:_flush():76] Configure stats pid to 1428650
2024-05-19 16:47:56,368 INFO    MainThread:1428650 [wandb_setup.py:_flush():76] Loading settings from /home/dan/.config/wandb/settings
2024-05-19 16:47:56,368 INFO    MainThread:1428650 [wandb_setup.py:_flush():76] Loading settings from /home/dan/mini_temporal/wandb/settings
2024-05-19 16:47:56,368 INFO    MainThread:1428650 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-19 16:47:56,368 INFO    MainThread:1428650 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-19 16:47:56,368 INFO    MainThread:1428650 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'training/fine_tune_gemma_halfway.py', 'program_abspath': '/home/dan/mini_temporal/training/fine_tune_gemma_halfway.py', 'program': '/home/dan/mini_temporal/training/fine_tune_gemma_halfway.py'}
2024-05-19 16:47:56,368 INFO    MainThread:1428650 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-19 16:47:56,368 INFO    MainThread:1428650 [wandb_init.py:_log_setup():527] Logging user logs to /home/dan/mini_temporal/wandb/run-20240519_164756-4z2l3bqi/logs/debug.log
2024-05-19 16:47:56,368 INFO    MainThread:1428650 [wandb_init.py:_log_setup():528] Logging internal logs to /home/dan/mini_temporal/wandb/run-20240519_164756-4z2l3bqi/logs/debug-internal.log
2024-05-19 16:47:56,368 INFO    MainThread:1428650 [wandb_init.py:init():567] calling init triggers
2024-05-19 16:47:56,368 INFO    MainThread:1428650 [wandb_init.py:init():574] wandb.init called with sweep_config: {}
config: {'dataset': 'TQE', 'model_context': 'no_context', 'batch_size': 32, 'base_model': 'gemma-1.1-2b-it', 'lr': 2e-05, 'epochs': 6}
2024-05-19 16:47:56,368 INFO    MainThread:1428650 [wandb_init.py:init():617] starting backend
2024-05-19 16:47:56,368 INFO    MainThread:1428650 [wandb_init.py:init():621] setting up manager
2024-05-19 16:47:56,369 INFO    MainThread:1428650 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-05-19 16:47:56,370 INFO    MainThread:1428650 [wandb_init.py:init():629] backend started and connected
2024-05-19 16:47:56,372 INFO    MainThread:1428650 [wandb_init.py:init():721] updated telemetry
2024-05-19 16:47:56,376 INFO    MainThread:1428650 [wandb_init.py:init():754] communicating run to backend with 90.0 second timeout
2024-05-19 16:47:56,449 WARNING MainThread:1428650 [wandb_init.py:init():1209] interrupted
Traceback (most recent call last):
  File "/home/dan/miniconda3/envs/danEnv/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 1187, in init
    run = wi.init()
          ^^^^^^^^^
  File "/home/dan/miniconda3/envs/danEnv/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 757, in init
    result = run_init_handle.wait(
             ^^^^^^^^^^^^^^^^^^^^^
  File "/home/dan/miniconda3/envs/danEnv/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dan/miniconda3/envs/danEnv/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dan/miniconda3/envs/danEnv/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dan/miniconda3/envs/danEnv/lib/python3.11/threading.py", line 629, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dan/miniconda3/envs/danEnv/lib/python3.11/threading.py", line 331, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2024-05-19 16:47:58,802 WARNING MsgRouterThr:1428650 [router.py:message_loop():77] message_loop has been closed
